adapter/integration-test/src/adapter-datasource-trigger.test.js

```diff
@@ -158,7 +157,7 @@ describe('Datasource triggering', () => {
       .post(`/datasources/${datasourceId}/trigger`)
       .send()
 
-    expect(triggerResponse.status).toBeGreaterThan(300) // request should fail (no 2xx status)
+    expect(triggerResponse.status).toBeGreaterThan(300)
```

Comment:
I don't think this comment is unnecessary. It is not immediately obvious that this assertion is meant to ensure that the request is supposed to fail, the comment makes it easier to understand this purpose

Reply:
Than the test description or the expect() method should reflect that behavior and not a comment.
IMHO it is a code smell if a comment next to a statement has to explain this statement.
For example
```
expect(triggerResponse.status).toBeEqual(NOT_FOUND)
```

If it is in a `runInBand` configuration
```

test('Should respond with NOT_FOUND for not existing XYZ ', () => {
expect(triggerResponse.status).toBeEqual(404)
})
```